BurnOutSharp/ProtectionType/Macrovision.cs

```diff
@@ -70,10 +97,17 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
         {
             // TODO: Add all common Macrovision directory path checks here
 
+            // Run C-Dilla directory checks
+            var cDilla = CDillaCheckDirectoryPath(path, files);
```

Comment:
**Question:** Were you planning on doing a list of detected protections similar to the PE checks?

Reply:
It doesn't seem as necessary to me compared to executable checks personally, and I was planning on doing it only if I came across a file where that would be needed.

Reply:
Since individual files might have multiple protections, then it makes sense if multiple files from those protections could be detected in a directory.